integration_test/validate_metadata_test.go

```diff
@@ -18,7 +20,42 @@ var agentMetricsMetadata []byte
 var thirdPartyDataDir embed.FS
 
 func TestValidateMetadataOfThirdPartyApps(t *testing.T) {
-	err := fs.WalkDir(thirdPartyDataDir, ".", func(path string, info fs.DirEntry, err error) error {
+	err := walkThirdPartyApps(func(contents []byte) error {
+		return validateMetadata(contents, &metadata.IntegrationMetadata{})
+	})
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestThirdPartyPublicUrls(t *testing.T) {
+	err := walkThirdPartyApps(func(contents []byte) error {
+		integrationMetadata := &metadata.IntegrationMetadata{}
+		err := validateMetadata(contents, integrationMetadata)
```

Comment:
You shouldn't need to validate it again.

Reply:
Changed in: [92b982e](https://github.com/GoogleCloudPlatform/ops-agent/pull/774/commits/92b982e27964acb2eda56ba3376d6661306efb55)